app/views/courses/index.html

```diff
@@ -0,0 +1,69 @@
+<div ng-controller='CoursesIndexCtrl as controller'>
+  <div class='courses-index-container'>
+    <!-- Page Heading -->
+    <div class='row'>
+      <div class='col-xs-12'>
+        <h1 translate='i18nCoursesPanelSection' class='page-header'></h1>
+      </div>
+    </div>
+    <!-- /Page Heading -->
+
+    <!-- Filter -->
+    <div class='filter-container'>
+      <div class='filter-fields'>
+        <div class='own-input-group'>
+          <span class='search-label' translate=i18nCourseId></span>
+          <input type='text' class='form-input-field' ng-model='controller.filter.courseId' placeholder="{{ 'i18nCourseId' | translate }}..."/>
+        </div>
+        <div class='own-input-group'>
+          <span class='search-label' translate='i18nCourseName'></span>
+          <input type='text' class='form-input-field' ng-model='controller.filter.courseName' placeholder="{{ 'i18nCourseName' | translate }}..."/>
+        </div>
+      </div>
+      <div class='filter-buttons'>
+        <button class='btn btn-default' type='submit'>
+          <span class='fa fa-search' aria-hidden='true'></span>
+          <span translate='i18nSearchButton' translate ></span>
+        </button>
+        <button class='btn btn-default' ng-click='controller.resetSearch()' type='submit'>
+          <span class='fa fa-repeat' aria-hidden='true'></span>
+          <span translate='i18nResetButton'></span>
+        </button>
+      </div>
+    </div>
+    <!-- /Filter -->
+
+    <!-- Content -->
+    <div class='table-responsive'>
+      <table class='table table-hover'>
+        <thead>
+          <tr>
+            <th translate='i18nCourseId'></th>
+            <th translate='i18nCourseName'></th>
+            <th translate='i18nCredits'></th>
+            <th translate='i18nSemester'></th>
+            <th translate='i18nActions'></th>
```

Comment:
This translation is not working OK. Check that the translation is' i18**n**Actions' at the translations file